docs/10-WebServices.md

```diff
@@ -67,6 +107,54 @@ class ApiHelper extends \Codeception\Module
 
 The same way you can receive request parameters and headers.
 
+### Validate JSON structures
+
+It is pretty common for API tests to not only validate the received data but to check the structure of the response. Response data is not something constant, and may change on each request, however the JSON/XML structure should kept the same for an API version. In order to check response structure REST module have some useful methods. 
+
+If we expect JSON response to be received we can check its structure with [JSONPath](http://goessner.net/articles/JsonPath/). It looks and sounds like XPath but is designed to work with JSON data, however we can convert JSON into XML and use XPath to validate the structure. Both approaches are valid and can be used in REST module:
```

Comment:
If we expect **a** JSON response to be received

and can be used in **the** REST module